### PR TITLE
Adding btw.so to the list

### PR DESCRIPTION
**Pull request by siddug**
## Add/Update/Remove <CompanyName>

- [x] I have read the [contributing guidelines](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md)
- [x] I agree to the [Code of Conduct](https://github.com/poteto/hiring-without-whiteboards/blob/master/CODE_OF_CONDUCT.md)
- [x] I have followed the [format](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md#format) prescribed in the contributing guidelines
- [ ] (OPTIONAL) In your pull request message, add additional context on the interview process if necessary

### CODE REVIEW

**Comment by SwapnilSoni1999** [comment]
It seems this site doesnt have any recruitment or hiring section. How to apply?

File: README.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ Also check out [No Whiteboards](https://nowhiteboards.io) to search for jobs at
 - [brightwheel](https://angel.co/brightwheel/jobs) | San Francisco, CA | Take home exercise, and systems design.
 - [Broad Institute's Data Sciences Platform](https://www.broadinstitute.org/data-sciences-platform) | Cambridge, MA | Phone screen, small take home project, both a technical and non-technical discussion panel, and a panel following up on the take home project walking through the solution and making a modification to the original code
 - [BrowserStack](https://www.browserstack.com/careers) | Mumbai, India; Dublin, Ireland | Initial phone screen with semi-technical questions, two 2-hour assignments based on real problems solved in company (allowed to lookup internet), final interview with EMs/Director.
+- [btw](https://www.btw.so) | Remote/ Singapore, Singapore / Bangalore, India | The open source platform to understand, engage and convert your visitors into customers
```
Comment (poteto) [inline README.md:147]: This should be a description of the interview process.

**Review by poteto — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (siddug) [reply README.md]: Thanks for the comment. It is updated now :)

**Comment by siddug** [comment]
Have updated the commit with the requested changes. Thanks @poteto

@SwapnilSoni1999 Do check the open roles here - https://angel.co/company/btw-2/jobs (Please do refer if you think anyone fits)

**Review by poteto — APPROVED** [review]

**Comment by poteto** [comment]
Thanks!

bors r+

**Pull request merged**